test/pantry-test.js

```diff
@@ -50,19 +51,33 @@ describe('Pantry', () => {
       }
     ]
 
-    pantry = new Pantry(ingredientsData, user1);
+    pantry = new Pantry(user1);
   });
 
   it('Should be a function', () => {
     expect(Pantry).to.be.a('function');
   });
 
-  it('Should be an instance of a pantry', () => {
+  it('Should be an instance of a Pantry', () => {
     expect(pantry).to.be.an.instanceOf(Pantry);
   });
+
+  it('Should be able to hold user data', () => {
+    expect(pantry.contents).to.equal(user1.pantry);
+  });
+
+  it('Should be able to hold user data', () => {
+    expect(pantry.user).to.equal(user1);
+  });
+
+  it('Should give an Error if a new Pantry has no user arguements', () => {
+    expect(() => { new Pantry() }).to.throw(Error);
+  });
```

Comment:
Lines 64-75 are tests for the needed properties.

Reply:
Same Comment as other test, Maybe we can have the method return null if no ingredients are given? or if we try to pass in other datatypes it could also return null